clients/include/testing_rotg.hpp

```diff
@@ -32,11 +32,10 @@ hipblasStatus_t testing_rotg(const Arguments& arg)
     host_vector<T> hs(1);
 
     // Initial data on CPU
-    srand(1);
-    hipblas_init<T>(ha, 1, 1, 1);
-    hipblas_init<T>(hb, 1, 1, 1);
-    hipblas_init<U>(hc, 1, 1, 1);
-    hipblas_init<T>(hs, 1, 1, 1);
+    hipblas_init_vector(ha, arg, 1, 1, 0, 1, hipblas_client_alpha_sets_nan, true);
```

Comment:
A very minor changez, could `arg` be renamed to `argus`? This would make it similar to all the other hipBLAS level 1 functions. Can be done in a separate PR.

Reply:
It would be good to change this across the testing, there are `arg`s and `argus`s sprinkled throughout the files. I'll make a PR addressing this after these changes go in for L1, L2, and L3.